python/ray/data/_internal/execution/interfaces.py

```diff
@@ -233,6 +233,10 @@ class TaskContext:
     # TODO(chengsu): clean it up from TaskContext with new optimizer framework.
     sub_progress_bar_dict: Optional[Dict[str, ProgressBar]] = None
 
+    # The underlying function called in a MapOperator; this is used when fusing
+    # an AllToAllOperator with an upstream MapOperator.
+    map_transform_fn: Optional["MapTransformFn"] = None
```

Comment:
can we call it `upstream_map_transform_fn` or `input_map_transform_fn`? We will soon add support to fuse downstream map operators later.